dist/vespa.spec

```diff
@@ -52,19 +52,21 @@ BuildRequires: gcc-toolset-11-gcc-c++
 BuildRequires: gcc-toolset-11-binutils
 BuildRequires: gcc-toolset-11-libasan-devel
 BuildRequires: gcc-toolset-11-libatomic-devel
+BuildRequires: python39-pybind11-devel
+BuildRequires: python39-devel
```

Comment:
The requires for vespa-ann-benchmark subpackage needs adjustment too.

Reply:
Adjusted